src/scripting/lua_bases.h

```diff
@@ -108,7 +108,9 @@ class LuaPlayerBase : public LuaBasesModuleClass {
 	 * Lua methods
 	 */
 	CLANG_DIAG_RESERVED_IDENTIFIER_OFF
```

Comment:
it seems like this Macro does not work so can it be deleted. maybe replacew by NOLINTBEGIN and NOLINTEND

Reply:
I don't know. It looks like it sets a compiler flag, so it looks different from NOLINT.